adagrams/game.py

```diff
@@ -1,13 +1,114 @@
 from random import randint
 
 def draw_letters():
-    pass
+    #returns an array of ten strings each containing one letter
+    #randomly drawn letters from pool of letters for the correct amount of letters available 
+    #this function should **not** change the pool of letters
+
+    LETTER_BANK_COUNT = {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2,
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1,
+        'Y': 2, 
+        'Z': 1
+    }
```

Comment:
This is nice work, using the python convention of using all caps to signal that these variables are not meant to be changed, constant variables. I would also suggest formatting longer dictionaries like this in the following manner for better readability. Take a look here [here](https://peps.python.org/pep-0008/#code-lay-out) to learn more about the Python style guide about and how to use multilines and indentation to create data structures.
```suggestion
LETTER_BANK_COUNT = {
    "A":9,"B":2,"C":2,"D":4,
    "E":12,"F":2,"G":3,"H":2,
    "I":9,"J":1,"K":1,"L":4,"M":2,
    "N":6,"O":8,"P":2,"Q":1,
    "R":6,"S":4,"T":6,"U":4,
    "V":2,"W":2,"X":1,
    "Y":2,"Z":1
}
```

Reply:
Thank you for this feedback. I did pull the list from the test and pasted into my code block, so I will def remember this for next time.